docs/api-docs/customers/customers-subscribers-overview.md

```diff
@@ -121,7 +121,7 @@ The V2 Customers API is comprised of customers, customer addresses and customer
 
 [Customer groups](https://developer.bigcommerce.com/api-reference/store-management/customers-v2/customer-groups/) allow you to organize your customers, give them discounts, and restrict access to specific products or categories. Customer groups are not yet available on the V3 Customers API.
 
-## Differences between V2 and V3 Customer API
+## Differences between V2 and V3 Customers API
 

```

Comment:
@ektravel, please add a transition sentence here. We want to avoid headers having no transitions into their subheaders.